library/Context/src/Juvix/Context/Types.hs

```diff
@@ -42,6 +39,28 @@ data From b
   | Outside b
   deriving (Show, Functor, Traversable, Foldable, Eq)
 
+data Info term ty sumRep = Info
+  { infoTable :: HashMap.T Symbol Sexp.T,
+    infoDef :: Definition term ty sumRep
+  }
+  deriving (Show, Read, Generic, Eq, NFData)
+
+data InfoRecord term ty sumRep = InfoRecord
```

Comment:
Will `InfoRecord` be removed in part 2 of this work? i.e can the `Record` (module) be encapsulated in the `Info` type?

Reply:
We'll still need it, as as the general `Info` type will still have a `def` which is still a variant.

```haskell
data Info = Term ... | Module ... | CurrentModule
```

thus the same trick is stil needed